test/test_cases/test_01_trivial.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DS xmlns="http://www.eprosima.com/XMLSchemas/discovery-server" user_shutdown="false">
+
+    <servers>
+        <server name="server" profile_name="UDP server" />
+    </servers>
+
+    <clients>
+        <client name="client1" profile_name="UDP_client1_server1">
+            <publisher topic="topic1"/>
+        </client>
+        <client name="client2" profile_name="UDP_client2_server1">
+            <subscriber topic="topic1"/>
+        </client>
+    </clients>
+
+    <snapshots file="./test_01_trivial.snapshot~">
+        <snapshot time="5">test_01_trivial_snapshot_1</snapshot>
+    </snapshots>
+
+    <profiles>
+        <participant profile_name="UDP_client1_server1" >
+            <rtps>
+                <prefix>63.6c.69.65.6e.74.31.5f.73.31.5f.5f</prefix>
```

Comment:
Adjust all GUID Prefixes so that participants in the same process are identified correctly, and so that participants in different processes do not attempt to use intra-process comms.